dataset/dataset.py

```diff
@@ -51,7 +51,7 @@ def __getitem__(self, index):
         img_path = os.path.join(self.path, 'images', img_file_name)
         lbl_path = os.path.join(self.path, 'labels', lbl_file_name)
         img = self.img_transformer(Image.open(img_path)).to(torch.float32).to(self.device)
-        lbl = self.lbl_transformer(Image.open(lbl_path)).to(torch.float32).to(self.device)[0]
+        lbl = self.lbl_transformer(Image.open(lbl_path)).to(torch.float32).long().to(self.device) 
```

Comment:
Possiamo risparmiarci di convertire le labels in torch.float32, io avevo provato e funziona anche senza